src/components/include/application_manager/hmi_capabilities.h

```diff
@@ -101,6 +101,19 @@ class HMICapabilities {
    */
   virtual const std::string& ccpu_version() const = 0;
 
+  /**
+   * @brief Interface used to store information about hardware version of the
+   * target
+   * @param hardware_version Received system/hmi hardware version
+   */
+  virtual void set_hardware_version(const std::string& hardware_version) = 0;
+
+  /**
+   * @brief Returns hardware version of the target
+   * @return TRUE if it supported, otherwise FALSE
```

Comment:
@ychernysheva it does not return true/false. it returns string

Reply:
Fixed in 0ae3584